v2/common/src/main/java/com/google/cloud/teleport/v2/cdc/mappers/MergeInfoMapper.java

```diff
@@ -152,12 +155,14 @@ public DataStreamPkCache getPkCache() {
                         allPkFields,
                         allSortFields,
                         METADATA_DELETED,
-                        String.format("%s.%s", // Staging Table
+                        String.format("%s.%s.%s", // Staging Table
+                                projectId,
```

Comment:
Please make sure that the arguments are aligned with each other. `projectId` in both cases is further indented than the other arguments.